kin-core/src/test/java/kin/core/PaymentWatcherTest.java

```diff
@@ -69,6 +69,8 @@ public Object answer(InvocationOnMock invocation) throws Throwable {
                     new Thread(new Runnable() {
                         @Override
                         public void run() {
+                            sleep();
```

Comment:
should separate the events into two different mocking scenarios, in our API we can't get both create account event and kin payment on the same stream

Reply:
why not? this is a better test - the transactions stream could include transactions other than payment or create account - this make sure that the filtering is correct (for both payment and create account operations). Also transaction stream that include create account and than Kin payment is a logical flow

Reply:
you're correct, my bad